gather-scheduler.js

```diff
@@ -90,10 +91,11 @@ class GatherScheduler {
       .then(traceContents => {
         tracingData.traceContents = traceContents;
       })
+      .then(_ => this.saveAssets(saveTrace, tracingData, url))
```

Comment:
maybe do what's done with e.g. `loadPage` above and do
```
.then(_ => {
  if (saveTrace) {
    this.saveAssets(tracingData, url)
  }
})
```
so `saveAssets` doesn't have to know about the conditional